### PR TITLE
fix app codesigning and notarization

### PR DESCRIPTION
**Pull request by yilmazbahadir**
* Fixed app code signing and notarization issues, updated the Readme
* Post-release ops: set the next release version

### CODE REVIEW

File: scripts/notarize.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const { notarize } = require('electron-notarize');
+var appBundleId = require('../package.json').build.appId;
```
Comment (Jaguar0625) [inline scripts/notarize.js]: var => const ?

File: scripts/notarize.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+
```
Comment (Jaguar0625) [inline scripts/notarize.js]: double blank line => single

File: scripts/notarize.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+// You will need to notarize the application if the "Developer ID Certificate" is first time being used
```
Comment (Jaguar0625) [inline scripts/notarize.js]: first time being used => being used for the first time

File: scripts/notarize.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+exports.default = async function notarizing(context) {
```
Comment (Jaguar0625) [inline scripts/notarize.js]: use lambda syntax?

File: scripts/notarize.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  const { electronPlatformName, appOutDir } = context;  
+  if (electronPlatformName !== 'darwin') {
+    console.log('Skipping notarizing because this is not a macOS build.');
```
Comment (Jaguar0625) [inline scripts/notarize.js]: notarizing => notarization (multiple)

File: scripts/notarize.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    return;
+  } else if (process.env.DESKTOP_APP_NOTARIZE !== 'true') {
+    console.log('Skipping notarizing because DESKTOP_APP_NOTARIZE env is not set.');
+    return;
+  } else if (process.env.DESKTOP_APP_APPLE_ID === undefined || process.env.DESKTOP_APP_APPLE_PASSWORD === undefined) {
+    console.log('Skipping notarizing because DESKTOP_APP_APPLE_ID or DESKTOP_APP_APPLE_PASSWORD env are not set.');
```
Comment (Jaguar0625) [inline scripts/notarize.js]: are => is

File: README.md [hunk 7]
```diff
@@ -54,9 +54,43 @@ If the circle next to `Node` is red, click on it and select another node from th
...
+npm run release
+</pre>
+
+7) Release for distribution: (Code signing for Apple builds - requires `Developer ID Certificate`)
+
+    7.1 On a MacOS machine, download the zip file containing the app signing certificates (ask team)
+
+    7.2 Extract the certificates and double click each one of them to add to the keychain (ask the team for private key password)
+    
+    7.3 If `Developer ID Certificate` is signing the app for the first time then the app needs to be notarized and this is an automated step in the process.You'll need to enable notarization by setting the following env vars.(skip this step already signed before)
```
Comment (Jaguar0625) [inline README.md]: spaces after periods
skip this step **if** already

File: README.md [hunk 8]
```diff
@@ -54,9 +54,43 @@ If the circle next to `Node` is red, click on it and select another node from th
...
+
+    <pre>export CSC_IDENTITY_AUTO_DISCOVERY=true</pre>
+
+    7.5 Run release
+    <pre>npm run release</pre>
+
+    7.6 Validate if the app is signed with a Developer ID Certificate
+    
+    <pre>spctl -a -t exec -v ./release/mac/Nem\ Wallet.app</pre>
+
```
Comment (Jaguar0625) [inline README.md]: double blank line => single

File: README.md [hunk 9]
```diff
@@ -54,9 +54,43 @@ If the circle next to `Node` is red, click on it and select another node from th
...
+    7.4 Enable auto discovery for code signing process to pick up the certificates from the keychain
+
+    <pre>export CSC_IDENTITY_AUTO_DISCOVERY=true</pre>
+
+    7.5 Run release
+    <pre>npm run release</pre>
+
+    7.6 Validate if the app is signed with a Developer ID Certificate
+    
+    <pre>spctl -a -t exec -v ./release/mac/Nem\ Wallet.app</pre>
```
Comment (Jaguar0625) [inline README.md]: maybe add comment about what successful output is ?

**Comment by yilmazbahadir** [comment]
@Jaguar0625 Thanks for reviewing, updated the PR, all of the feedbacks should be fixed now.

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**